src/components/inspector/controls/TextControl.tsx

```diff
@@ -33,7 +33,8 @@ const TextControl: React.FC<TextControlPropsType> = ({
         size="sm"
         value={value || ''}
         type="text"
-        width={hasColumn ? '3rem' : '100%'}
+        // width={hasColumn ? '10rem' : '100%'}
```

Comment:
This should probably work while keeping this right ?

Reply:
I tried it and it worked correctly, I will change it.

Reply:
I fix it on https://github.com/premieroctet/openchakra/pull/131/commits/59f221bec1be0e7627a011fddd1380b591fe0823#diff-b9ac78e00ce182ec5001f330c2ae52942dd9c9e90dbb619125ada2384ed5cd73